pgmq-rs/src/pg_ext.rs

```diff
@@ -104,8 +104,7 @@ impl PGMQueueExt {
         // we need to check whether the queue exists first
         // pg_partman create operations are currently unable to be idempotent
         let exists_stmt = format!(
-            "SELECT EXISTS(SELECT * from part_config where parent_table = '{queue_table}');",
-            queue_table = queue_table
+            "SELECT EXISTS(SELECT * from part_config where parent_table = '{queue_table}');"
```

Comment:
since we are in here cleaning this up, can we use a bind parameter instead of string formatting?

Reply:
Sure! Let me know if [59bdf9b](https://github.com/tembo-io/pgmq/pull/341/commits/59bdf9ba5cf6bf58da3c5565b48c13c8fab9cdf0) isn't what you had in mind.

Reply:
Yes that is exactly what I had in mind. Thank you!

Reply:
You're welcome! I'm happy to help.